.pre-commit-config.yaml

```diff
@@ -96,6 +96,7 @@ repos:
         # Excluding Example DAGs temporarily until they can be fixed
         exclude: ^tests/.*\.py$|.*example_dags/.*
         args: ["-c", "./pyproject.toml"]
+        additional_dependencies: [".[toml]"]
```

Comment:
why this change?

Reply:
Somehow, I could no longer commit without this. I was getting this error:
```
pyproject.toml : toml parser not available, reinstall with toml extra
```

I followed the Stackoverflow answer https://stackoverflow.com/questions/75303927/pyproject-toml-toml-parser-not-available-reinstall-with-toml-extra

Reply:
hmm, docs suggest something like https://bandit.readthedocs.io/en/latest/config.html?highlight=additional_dependencies#bandit-settings
```
additional_dependencies: ["bandit[toml]"]
```
not sure if it will work in this case